cirq/google/gate_sets.py

```diff
@@ -13,5 +13,115 @@
+            ]),
+        op_serializer.GateOpSerializer(gate_type=ops.ZPowGate,
+                                       serialized_gate_id='exp_z',
+                                       args=[
+                                           op_serializer.SerializingArg(
+                                               serialized_name='half_turns',
+                                               serialized_type=float,
+                                               gate_getter='exponent')
+                                       ]),
+        op_serializer.GateOpSerializer(
+            gate_type=ops.XPowGate,
+            serialized_gate_id='exp_w',
+            args=[
+                op_serializer.SerializingArg(serialized_name='axis_half_turns',
+                                             serialized_type=float,
+                                             gate_getter=lambda x: 0.0),
+                op_serializer.SerializingArg(serialized_name='half_turns',
+                                             serialized_type=float,
+                                             gate_getter='exponent'),
+            ]),
+        op_serializer.GateOpSerializer(
+            gate_type=ops.YPowGate,
+            serialized_gate_id='exp_w',
+            args=[
+                op_serializer.SerializingArg(serialized_name='axis_half_turns',
+                                             serialized_type=float,
+                                             gate_getter=lambda x: 0.5),
+                op_serializer.SerializingArg(serialized_name='half_turns',
+                                             serialized_type=float,
+                                             gate_getter='exponent'),
+            ]),
+        op_serializer.GateOpSerializer(gate_type=ops.CZPowGate,
+                                       serialized_gate_id='exp_11',
+                                       args=[
+                                           op_serializer.SerializingArg(
+                                               serialized_name='half_turns',
+                                               serialized_type=float,
+                                               gate_getter='exponent')
+                                       ]),
+        op_serializer.GateOpSerializer(
+            gate_type=ops.MeasurementGate,
+            serialized_gate_id='meas',
+            args=[
+                op_serializer.SerializingArg(
+                    serialized_name='key',
+                    serialized_type=str,
+                    gate_getter=protocols.measurement_key),
+                op_serializer.SerializingArg(serialized_name='invert_mask',
+                                             serialized_type=List[bool],
+                                             gate_getter='invert_mask')
+            ])
+    ],
+    deserializers=[
+        op_deserializer.GateOpDeserializer(
+            serialized_gate_id='exp_w',
+            gate_constructor=ops.PhasedXPowGate,
```

Comment:
I does because the constructor for PhasedXPowGate returns gates of those types as appropriate.  But XPowGate is not a subclass of PhasedXPowGate, so to get the serialization to work you need the serializer cases.